File: programme/feet-to-inches/feet-to-inches.go
// Write a program to convert feet to inches

package main

import "fmt"

func feetToInches(feet float32) float32 {
	return feet * 12.0
}

func main() {
	fmt.Print("Input  : ")

	var feet float32
	fmt.Scan(&feet)

	fmt.Println("Output :", feetToInches(feet))
}

File: programme/feet-to-inches/FeetToInches.java
// Write a programme to convert feet to inches
import java.util.Scanner;

class FeetToInches {

  public static void main(String[] args) {
    try (Scanner sc = new Scanner(System.in)) {
      System.out.print("Input  : ");
      int feet = sc.nextInt();
      System.out.print("Output : ");
      System.out.println(feet * 12);
    } catch (Exception e) {
      System.err.println("An Error occured");
    }
  }
}
